ci-operator/jobs/openshift/cluster-config-operator/openshift-cluster-config-operator-master-presubmits.yaml

```diff
@@ -72,6 +72,78 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn,?($|\s.*)
+  - agent: kubernetes
```

Comment:
where do these come from?  Are they generated?

Reply:
Everything in the `presubmits` file came from `make jobs`, I haven't touched it